### PR TITLE
maven_install enablement

### PR DESCRIPTION
**Pull request by btbuxton**
I think this should be put into it's branch (maven_install). So, DO NOT MERGE.

Looking for feedback before I create the new branch.

### CODE REVIEW

File: crawl/dependency.py [hunk 1]
```diff
@@ -133,6 +133,16 @@ def __repr__(self):
...
+    @classmethod
+    def from_coordinates(cls, bazel_label_name, coord_str):
+        parts = coord_str.split(':')
+        if len(parts) == 3:
+            return cls(bazel_label_name, parts[0], parts[1], parts[2])
+        if len(parts) == 4:
+            return cls(bazel_label_name, parts[0], parts[1], parts[3], scope=parts[2])
+        if len(parts) == 5:
+            return cls(bazel_label_name, parts[0], parts[1], parts[4], scope=parts[2], classifier=parts[3])
+        raise Exception('Illegal maven coordinate: %s' % coord_str)
```
Comment (simontoens) [inline crawl/dependency.py:145]: Is this similar to new_dep_from_maven_art_str below?  Can we remove one of those methods?  Docstring?

**Comment by simontoens** [comment]
Looks good

File: crawl/dependency.py (hunk 1 above)
Comment (simontoens) [reply crawl/dependency.py:145]: Also probably good to add a test for this method specifically?

Comment (btbuxton) [reply crawl/dependency.py:145]: Didn't see the new_dep_from_maven_art_str - I will remove the class method

**Comment by btbuxton** [comment]
Closing. Will create new branch maven_install with this code.

**Pull request closed without merging**